src/Helper/SerializationHelper.php

```diff
@@ -142,8 +142,22 @@ public static function getProperties($obj)
             $rc = new ReflectionClass($obj);
             do {
                 $rp = array();
+
+                // Note that we will not include any properties in traits
+                $traits = $rc->getTraits();
+                $skipProperties = array();
+                if (!empty($traits)) {
+                    foreach ($traits as $trait) {
+                        foreach ($trait->getProperties() as $traitProperty) {
+                            $skipProperties[] = $traitProperty->getName();
+                        }
+                    }
+                }
                 /* @var $p \ReflectionProperty */
                 foreach ($rc->getProperties() as $p) {
+                    if (in_array($p->getName(), $skipProperties)) {
```

Comment:
You're comparing only strings here, it's safe to use strict comparison here.
```suggestion
                    if (in_array($p->getName(), $skipProperties, true)) {
```